Makefile

```diff
@@ -1,5 +1,8 @@
 .PHONY: test docs
 
+COVERAGE_OPTS := --rcfile $(CURDIR)/.coveragerc
```

Comment:
i thought we only needed to specify this for `pytest-cov` (it's already the default). do we definitely need this explicitly?